client/app/telecom/telephony/billingAccount/administration/optionsGroup/telecom-telephony-billing-account-administration-options-group.html

```diff
@@ -16,134 +9,57 @@ <h1 data-translate="telephony_options_group_title"></h1>
-                                           data-uib-btn-radio="false"
-                                           data-translate="common_no_capitalized">
-                                    </label>
-                                </div>
-                            </div>
-                        </div>
-                    </div> <!-- END OF HIDDEN EXTERNAL NUMBER -->
-                    <!-- /.widget-presentation -->
+        <oui-back-button data-title="{{:: 'telephony_options_group_title' | translate }}"
+                         data-href="OptionsGroupCtrl.$state.href('telecom.telephony.administration')">
+        </oui-back-button>
 
-                    <!-- OVERRIDE DISPLAYED NUMBER -->
-                    <div class="widget-presentation">
-                        <header class="widget-presentation-header">
-                            <h2 class="widget-presentation-title"
-                                data-translate="telephony_billing_account_administration_options_group_overrideDisplayedNumber">
-                            </h2>
-                        </header>
-                        <div class="widget-presentation-content">
-                            <div class="oui-box mb-3"
-                                 data-translate="telephony_billing_account_administration_options_group_overrideDisplayedNumber_info">
-                            </div>
-                            <div class="form-group">
-                                <label for="overrideDisplayedNumber"
-                                       class="control-label"
-                                       data-translate="telephony_billing_account_administration_options_group_overrideDisplayedNumber_label">
-                                </label>
-                                <div class="btn-group d-block clearfix">
-                                    <label class="btn btn-default"
-                                           data-ng-model="OptionsGroupCtrl.optionsGroupForm.telephony.overrideDisplayedNumber"
-                                           data-uib-btn-radio="true"
-                                           data-translate="common_yes_capitalized">
-                                    </label>
-                                    <label class="btn btn-default"
-                                           data-ng-model="OptionsGroupCtrl.optionsGroupForm.telephony.overrideDisplayedNumber"
-                                           data-uib-btn-radio="false"
-                                           data-translate="common_no_capitalized">
-                                    </label>
-                                </div>
-                            </div>
-                        </div>
-                    </div> <!-- END OF OVERRIDE DISPLAYED NUMBER -->
-                    <!-- /.widget-presentation -->
+        <form name="OptionsGroupCtrl.optionsGroupSettings"
+              data-ng-submit="OptionsGroupCtrl.changeSettings()">
```

Comment:
I know there is no validation in this form but to standardize and create a clean copy/paste example, you should check if form is $valid before submitting. I saw self.optionsGroupSettings.$invalid in the controller but it's not standard with other integrations